api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -436,11 +436,18 @@ message Cluster {
   }
 
   // Common configuration for all load balancer implementations.
-  // [#next-free-field: 8]
+  // [#next-free-field: 9]
   message CommonLbConfig {
     option (udpa.annotations.versioning).previous_message_type =
         "envoy.api.v2.Cluster.CommonLbConfig";
 
+    // Refer to :ref:`endpoint warming policy types <arch_overview_load_balancing_slow_start_endpoint_warming_policy_types>`
+    // for an explanation on each policy type.
+    enum EndpointWarmingPolicy {
+      NO_WAIT = 0;
+      WAIT_FOR_FIRST_PASSING_HC = 1;
+    }
```

Comment:
inline this into `SlowStartConfig`?

Reply:
think the initial suggestion was to reuse this message type across other mechanisms in Envoy. We could make it top level message later if it will be actually used anywhere else.